boilerplate.js

```diff
@@ -202,11 +202,12 @@ async function install (context) {
       })
     }
 
-    if (parameters.options.lint !== 'false') {
-      await system.spawn(`ignite add standard@"~>1.0.0" ${debugFlag}`, {
-        stdio: 'inherit'
-      })
-    }
+    // TODO: Add Airbnb
+    // if (parameters.options.lint !== 'false') {
```

Comment:
I don't mind switching the linter, but there needs to be one. This commented out code is what is failing the build and it took me a while to figure out what was going wrong. 😬 

Also this repo's linter should match so if the boilerplate switched from `standard` [this should](https://github.com/jbosse/ignite-expo-boilerplate/blob/e604d7700dc8902bc029eb1504808a5e7eea7dd7/package.json#L50), too.